File: downloads/calculator/calculator.cs
using System;

class Calculator {
    static void Main() {
        Console.WriteLine("Simple C# Calculator");
        while (true) {
            Console.Write("Enter expression (or 'exit' to quit): ");
            string input = Console.ReadLine();
            if(input.ToLower() == "exit") break;
            try {
                var result = new System.Data.DataTable().Compute(input, null);
                Console.WriteLine("Result: " + result);
            } catch {
                Console.WriteLine("Invalid input.");
            }
        }
    }
}

File: downloads/calculator/calculator.py
# Python console calculator
def calculator():
    print("Simple Python Calculator")
    while True:
        try:
            expr = input("Enter expression (or 'exit' to quit): ")
            if expr.lower() == "exit":
                break
            print("Result:", eval(expr))
        except Exception as e:
            print("Error:", e)

if __name__ == "__main__":
    calculator()
